src/base/rss/rss_feed.cpp

```diff
@@ -96,7 +99,6 @@ Feed::Feed(const QUuid &uid, const QString &url, const QString &path, Session *s
 Feed::~Feed()
 {
     emit aboutToBeDestroyed(this);
-    Utils::Fs::forceRemove(m_iconPath);
```

Comment:
Isn't it now necessary to delete icon when deleting a Feed? I.e. when user removes Feed from list.